cypress/page/iframe.page.ts

```diff
@@ -0,0 +1,25 @@
+class IframePage {
+    private testPage: string;
+    private iframe: string;
+    private title: string;
+  
+    constructor() {
+      this.testPage = 'https://www.w3schools.com/html/html_iframe.asp';
+      this.iframe = '[src="default.asp"]';
+      this.title = 'a[title="CSS Tutorial"]';     
+    }
+
+    public visit(): void{
+        cy.visit(this.testPage);
+    }
+
+    public getFrameTitle(tittlePage: string): void{
+        return cy.iframe(this.iframe).should("contain.text", tittlePage);
+    }
+      
+    public goToCssPageInFrame(): void{
+        cy.iframe(this.iframe).find(this.title).click();
```

Comment:
probably, you will need also to add call the command frameLoaded. If the things start failing try to add this:
```suggestion
        cy.iframe(this.iframe).find(this.title).click();
        cy.frameLoaded(this.iframe, {url: urlToCssPage});
```

Reply:
resolved

Reply:
@holgiosalos Do you know how I can solve this error?
Local:
![image](https://user-images.githubusercontent.com/33491290/182722623-268b30be-2ba2-4cbc-84ba-aa1f15e86132.png)
CI:
![image](https://user-images.githubusercontent.com/33491290/182722713-a346a194-74e3-4824-9b59-52b4514ddbdc.png)

Reply:
i solved it, thanks!

Reply:
yes, the guide has something that you can do, to fix it

Reply:
sorry, it has not that part, i should add it